File: platform/sdk/nodejs/examples/graph/graph-015_proj_type_out_1.js
const { GraphClient } = require("../../xuanji-sdk-graph");
const client = new GraphClient({ endpoint: "graph.local" });
const result = client.projTypeOut1("node-abc");
if (!result.ok || result.projectType !== "GRAPH") process.exit(1);
console.log("XJ-OK: graph-015_proj_type_out_1");
process.exit(0);

File: platform/sdk/python/examples/graph/graph-015_proj_type_out_1.py
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from mox_sdk_graph import GraphClient

client = GraphClient({"endpoint": "graph.local"})
result = client.projTypeOut1("node-abc")
if not result["ok"] or result["projectType"] != "GRAPH":
    sys.exit(1)
print("XJ-OK: graph-015_proj_type_out_1")
sys.exit(0)
